integrated_channels/integrated_channel/tasks.py

```diff
@@ -149,28 +157,32 @@ def transmit_single_subsection_learner_data(username, course_run_id, subsection_
     """
     start = time.time()
     user = User.objects.get(username=username)
-    enterprise_customer = get_enterprise_customer_for_user(user)
+    enterprise_uuids = get_enterprise_uuids_for_user_and_course(user, course_run_id, active=True)
     channel_utils = IntegratedChannelCommandUtils()
 
-    # Transmit the learner data to each integrated channelStarting Export
-    for channel in channel_utils.get_integrated_channels(
-            {'channel': None, 'enterprise_customer': enterprise_customer.uuid}
-    ):
-        integrated_channel = INTEGRATED_CHANNEL_CHOICES[channel.channel_code()].objects.get(pk=channel.pk)
-        integrated_channel.transmit_single_subsection_learner_data(
-            learner_to_transmit=user,
-            course_run_id=course_run_id,
-            grade=grade,
-            subsection_id=subsection_id
-        )
-
-    duration = time.time() - start
-    LOGGER.info(
-        '[Integrated Channel] Single learner data transmission task finished.'
-        ' Course: {course_run}, Duration: {duration}, Username: {username}'.format(
-            username=username,
-            course_run=course_run_id,
-            duration=duration))
+    # Transmit the learner data to each integrated channel for each related customer.
+    # Starting Export. N customer is usually 1 but multiple are supported in codebase.
+    for enterprise_customer_uuid in enterprise_customer_uuids:
```

Comment:
is this supposed to be `enterprise_uuids`?

Reply:
So... the data model is EnterpriseCustomer. Buuut... pylint complained it was too long when I named the function get_enterprise_customer_uuids_for_user_and_course.  Suggestions welcome?